src/get-started/java-binding.md

```diff
@@ -9,21 +9,46 @@ To run a node with your Java service you need to use Exonum Java application.
 You can download an archive containing the application and all the necessary
 dependencies on [the Releases page][github-releases] on GitHub. We suggest using
 `debug` version during development and `release` for deployment.
-Follow these steps:
 
-- Install [Maven 3][maven-install] which is essential for developing and building
-  Java service.
-- Install [Libsodium][libsodium] development library for your system.
-  You can use the following command for Ubuntu:
+### Mac OS
+
+For Mac users, we provide a [Homebrew][homebrew] repository, which gives the
+easiest way of installing Exonum Java App:
+
+```bash
+brew tap exonum/exonum
+brew install exonum-java
+```
+
+This will install `exonum-java` binary with all the necessary dependencies.
+However, you still need to install [Maven 3][maven-install] and follow the
+steps mentioned in [After Install][#after-install] section below.
+
+You can also install Exonum Java App manually:
+
+- Install [Libsodium][libsodium] and RocksDB libraries:
 
   ```bash
-  sudo apt-get update && sudo apt-get install libsodium-dev
+  brew install libsodium rocksdb
   ```
 
-  For Mac OS, you can use [Homebrew][brew-install] as follows:
+- Download and unpack the archive from the [Releases page][github-releases]
+  into some known location. To install the latest release to `~/bin`:
 
   ```bash
-  brew install libsodium rocksdb
+  mkdir -p ~/bin
+  cd ~/bin
+  unzip /path/to/downloaded/exonum-java-0.6.0-release.zip
+  ```
+
```

Comment:
Shall we duplicate "build from sources" option in Mac OS and Linux — now it is in `After Install`, which does not make sense.

Or update the intro (it needs that anyway, because it unconditionally refers to Downloads), that there are three opertions:
* (macOS only) Install from brew
* Install pre-built binaries from Github releases
* Build from source

?

Reply:
I like the idea of making a common intro. First describe all the possible installation options. Then describe steps of each option separately. Options that are common for different OS's should be described once with remarks as to OS differences.
I can help in reorganizing the document. I will need to discuss the installation flow with somebody from the team, though.
As for grammar the PR looks ok to me.